E2E-tests/tests/committee/test_committee.py

```diff
@@ -35,6 +35,44 @@ def calculate_d_param_tolerance(pc_epochs_in_mc_epoch, d_param_p, d_param_t):
 
 class TestCommitteeDistribution:
 
+    @mark.committee_distribution
+    @mark.ariadne
+    @mark.probability
+    def test_update_d_param(
+        self,
+        api: BlockchainApi,
+        config: ApiConfig,
+        current_mc_epoch,
+    ):
+        """
+        * get latest (n + 1) mc epoch ariadne parameters
+        * run update d param
+        * get next (n + 2) mc epoch ariadne parameters
+        * confirm that d param was updated
+        """
+        data = api.get_ariadne_parameters(current_mc_epoch + 1)
+
+        d_param = data["dParameter"]
+        current_p = d_param["numPermissionedCandidates"]
+        current_t = d_param["numRegisteredCandidates"]
+        new_p = 0
+        new_t = 0
+        p_floor = config.nodes_config.permissioned_min_ariadne_d_param_value
+        p_ceil = config.nodes_config.permissioned_max_ariadne_d_param_value
+        t_floor = config.nodes_config.trustless_min_ariadne_d_param_value
+        t_ceil = config.nodes_config.trustless_max_ariadne_d_param_value
+        while new_p + new_t == 0 or (current_p == new_p and current_t == new_t):
+            new_p = np.random.randint(p_floor, p_ceil)
+            new_t = np.random.randint(t_floor, t_ceil)
+
+        api.update_d_param(new_p, new_t)
```

Comment:
Please remind me, if we do updateDParam with the same values that exist, will the command execute or return an error? I'm worried about your assertion on line 74.
Maybe we should skip the test if the range is only 1 value AND it matches the existing value?

Reply:
yeah I believe it errors out when you try to update it with the same values - I'll confirm

Reply:
Actually just confirmed that we have a rescue around the pc_contract_cli command. But it would normally fail. I was going to add a skip, but looks like we don't need it. What do you think?

```
        try:
            response = self.handle_response(result)
        except SidechainMainCliException as e:
            if 'InvalidCLIParams "Provided values have already been set. Please check."' in str(e):
                response = {}
            else:
                raise e
```